db.go

```diff
@@ -228,7 +236,17 @@ func openDialector(dialect gorm.Dialector, opt ...Option) (*DB, error) {
 		}
 		underlyingDB.SetMaxOpenConns(opts.WithMaxOpenConnections)
 	}
-	return &DB{wrapped: db}, nil
+
+	ret := &DB{wrapped: db}
+	ret.Debug(opts.WithDebug)
+	return ret, nil
+}
+
+// LogWriter defines an interface which can be used when passing a logger via
+// WithLogger(...).  This interface allows callers to override the default
+// behavior for a logger (the default only emits postgres errors)
+type LogWriter interface {
+	Printf(string, ...interface{})
```

Comment:
Nit: do we use `any` in this codebase yet?
```suggestion
	Printf(string, ...any)
```

Reply:
ty